### ISSUE TITLE
Layout YAML files

### ISSUE THREAD

**Issue by JoomFX** [gantry/gantry5#125]
It is not a defect but a question or an enhancement.

Guys, why do you have two completely different syntax/structures for the Layout YAML files? Wouldn't it be better to use just the friendlier one that you can see in the `_joomla_-_protostar.yaml` file in `media/gantry5/engines/nucleus/layouts` (instead of the one you have in `/hydrogen/layouts/` )?

In the simple (friendlier) one you also define what particles/atoms should be assigned by default.
From what I can see, the only thing you cannot do in the simple/friendlier syntax is to have the custom CSS classes and Attributes assigned by default.

Anyway, I'm almost sure that it is not that simple and that I'm missing the logic here, but still I wanted to ask this question :)

**Comment by JoomFX** [comment]
hmm, now when I read the comment from @w00fz in the Gitter chat, I released that he actually explained most of it:

*The big fat yaml file you see is a translation of the json structure used for the admin to build reconstruct and manage the layout in admin and for the renderer in frontend
What you should be modifying is the presets yaml that have a more user friendly content
The generated ones like home or default should be modified if you have attributes you want to change fast without padding through the admin*

But then, where are the original layout files for "Home" and "Default"? The ones in `/hydrogen/layouts/` are the already translated ones, so where are the originals?

Well, it is confusing indeed :)

**Comment by mahagr** [comment]
Originally we had only json format, but as it was so hard to read, I decided to add a simple way to generate layout in yaml. It was ment for our theme developers to allow them quickly make rough version of the layout, but it had some extensions later to add the sized.

The simple version cannot do many things that are possible in the real layout, so it was only meant to get designer started and to build the basic structure. We do have some plans to extend it, but those are not high in the priority lists as after we got rid of json in favour of yaml, the file became slightly easier to read.
